agent/ecscni/types.go

```diff
@@ -124,6 +129,28 @@ type ENIConfig struct {
 	SubnetGatewayIPV4Address string `json:"subnetgateway-ipv4-address"`
 }
 
+// AppMeshConfig contains all the information needed to invoke the app mesh plugin
+type AppMeshConfig struct {
+	// Type is the cni plugin name
+	Type string `json:"type,omitempty"`
+	// CNIVersion is the cni spec version to use
+	CNIVersion string `json:"cniVersion,omitempty"`
+	// Egress traffic from the processes owned by the UID will be ignored
+	IgnoredUID string `json:"ignoredUID,omitempty"`
```

Comment:
Commentary for these fields is still not following golang convention.

Reply:
My bad, updated few configs from earlier comment, missed this config. Updated.